input: 'city.png'
depth_image: ''
min_transmission: 0.2
max_atm_light: 220
window: 15
filter_radius: 40
underwater_dehaze: False

